frontend-react/src/components/Table/Table.tsx

```diff
@@ -147,6 +154,20 @@ const Table = ({ config, filterManager, cursorManager }: TableProps) => {
                         : displayValue}
                 </NavLink>
             );
+        } else if (columnConfig.actionable) {
+            const { action, param } = columnConfig.actionable;
+            const doAction = () => {
```

Comment:
Turns out we needed action columns for the History table, so went ahead and added that functionality.